core/iwasm/compilation/aot_emit_numberic.c

```diff
@@ -234,15 +234,49 @@ compile_op_float_min_max(AOTCompContext *comp_ctx, AOTFuncContext *func_ctx,
         nan = LLVMConstRealOfString(ret_type, "NaN");
     char *intrinsic = is_min ? (is_f32 ? "llvm.minnum.f32" : "llvm.minnum.f64")
                              : (is_f32 ? "llvm.maxnum.f32" : "llvm.maxnum.f64");
+    bool is_i32 = is_f32;
```

Comment:
Seems that `is_i32` isn't used?

Reply:
Emm, is_i32 used in macro `LLVM_BUILD_OP_OR_INTRINSIC`

Reply:
Got it, thanks.